src/OpenTelemetryEngine/ResponseObjects/GetActivityResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Diagnostics;
+
+namespace OpenTelemetryEngine.ResponseObjects
+{
+    public class GetActivityResponse
```

Comment:
Do we have any specific plans to add anything else to this class in the foreseeable future? Just wondering if we could return `Activity?` from `GetActivityForInvocation` directly, so this class is unnecessary. (This is just a nitpick, not blocking.)